src/routes/web.php

```diff
@@ -58,3 +61,13 @@
     SigninController::class,
     'login'
 ]);
+
+Route::get('/posts-page', [
+    UserPostController::class,
+    'showPostsPage'
+]);
+
+Route::post('/create-post', [
+    UserPostController::class, 
+    'create'
+]);
```

Comment:
I noticed there's no middleware on these. Does this mean even unauthenticated user can post and view posts?

Reply:
Unauthenticated users must not view or create posts. I will add middleware.